### PR TITLE
Add log level mapping for ChromeDriver

### LINKED ISSUE #7493: Add log level mapping for ChromeDriver just like for Firefox
ChromeDriver uses the following log levels:

ALL, DEBUG, INFO, WARNING, SEVERE, and OFF

See https://cs.chromium.org/chromium/src/chrome/test/chromedriver/logging.cc

Selenium uses:

"OFF", "SEVERE", "WARNING", "INFO", "CONFIG", "FINE", "FINER", "FINEST", "ALL".

https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#loggingpreferences-json-object

Please add a mapping between the two list like for Firefox here: https://github.com/SeleniumHQ/selenium/blob/master/java/client/src/org/openqa/selenium/firefox/FirefoxDriverLogLevel.java

Chromium is probably also affected.

The same needs to be done for the Python implementation of Selenium.

Related issue: https://bugs.chromium.org/p/chromedriver/issues/detail?id=457#c10

### PR DESCRIPTION
**Pull request by bongosway**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

This will enable setting ChromeDriver log level via ChromeOption.

This removes the need for this line of code `System.setProperty("webdriver.chrome.verboseLogging", true)`

Set Logging like so:

`ChromeOptions options = new ChromeOptions(); `
`options.setLogLevel(ChromeDriverLogLevel.ALL)`

### Motivation and Context

Fixes #7493
Makes adapting to future w3c spec on logging easy to manage.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by bongosway** [comment]
This [page](https://sites.google.com/a/chromium.org/chromedriver/logging) will need updating, how do I go about that?

Thanks.

File: java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java [hunk 1]
```diff
@@ -56,13 +56,6 @@
   public static final String CHROME_DRIVER_APPEND_LOG_PROPERTY =
       "webdriver.chrome.appendLog";
 
-  /**
-   * Boolean system property that defines whether the chromedriver executable should be started
-   * with verbose logging.
-   */
-  public static final String CHROME_DRIVER_VERBOSE_LOG_PROPERTY =
```
Comment (shs96c) [inline java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java:63]: This is part of the public API of the language binding, so it's not safe to remove this without deprecating it first. I think the best way to handle this is to check for a `CHOME_DRIVER_LOG_LEVEL_PROPERTY` and have that take precedence if set.

File: java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java [hunk 2]
```diff
@@ -162,11 +169,11 @@ public Builder withAppendLog(boolean appendLog) {
     /**
      * Configures the driver server verbosity.
      *
-     * @param verbose True for verbose output, false otherwise.
+     * @param logLevel {@link ChromeDriverLogLevel} for desired log level output.
      * @return A self reference.
      */
-    public Builder withVerbose(boolean verbose) {
-      this.verbose = verbose;
```
Comment (shs96c) [inline java/client/src/org/openqa/selenium/chrome/ChromeDriverService.java:169]: `withVerbose` is a general API that all DriverServiceBuilders expose. Best to have this delegate down to `withLogLevel` and have it set to `DEBUG` or `ALL`. That way, people trying to do generalised set up get what they expect, and power users can do something more fine-grained.

**Comment by bongosway** [comment]
@shs96c requested changes have been made. Kindly review at your convenience.

Many thanks

**Comment by shs96c** [comment]
Merged into the tree. Thank you once again!

**Pull request merged**